shared/chat/conversation/messages/attachment/index.js

```diff
@@ -61,11 +61,13 @@ function PreviewImage({
       marginTop: globalMargins.xtiny,
       position: 'relative',
     }
+
+    const imgWidth = imgMaxWidth()
     const imgStyle = {
       borderRadius: 4,
       ...(previewSize
-        ? {height: previewSize.height, width: previewSize.width}
-        : {maxHeight: 320, maxWidth: 320}),
+        ? {height: previewSize.height, width: Math.min(previewSize.width, imgWidth)}
```

Comment:
Don't you need to scale the `previewSize.height` proportionally to the width? Since both width and height are specified, won't it distort images?